### PR TITLE
Huda3

### PR DESCRIPTION
**Pull request by kpslp**
@SenthilKumarSivakami

### CODE REVIEW

File: notifications.md [hunk 1]
```diff
@@ -1,60 +1,20 @@
 # Notifications
 
-Notifications in TechBiz convey [information](#informational) to users and also [alert](#alerts) users to tasks awaiting their action.
+Notifications in TechBiz convey information and alert users to tasks awaiting their action. Users can receive notifications through email and Techbiz portal.
```
Comment (SenthilKumarSivakami) [inline notifications.md:3]: Users receive notifications by email and on the TechBiz portal.

Please take note of the brand name "TechBiz"

File: notifications.md [hunk 2]
```diff
@@ -1,60 +1,20 @@
...
 
-Click on each topic to know more:
+>**Note:** Alert notifications are specific to a user role.
```
Comment (SenthilKumarSivakami) [inline notifications.md:5]: what do we mean by this? Can you pls elaborate more for us to understand better?

File: notifications.md [hunk 3]
```diff
@@ -1,60 +1,20 @@
 # Notifications
 
-Notifications in TechBiz convey [information](#informational) to users and also [alert](#alerts) users to tasks awaiting their action.
```
Comment (SenthilKumarSivakami) [inline notifications.md:3]: is that information? I think it is a kind of status update of user requests. If you want, please confirm with the team.

File: notifications.md (hunk 1 above)
Comment (SenthilKumarSivakami) [inline notifications.md:3]: is that information? I think it is a kind of status update of user requests. If you want, please confirm with the team.

File: notifications.md [hunk 4]
```diff
@@ -1,60 +1,20 @@
...
-
-
-## Complete tasks
-
-**Prerequisites**
-
-- User has received an alert notification from TechBiz.
-
-To act on an alert, a user needs do as follows:
+## Accessing notifications on portal
```
Comment (SenthilKumarSivakami) [inline notifications.md:8]: why should this be a heading 2, and are the steps below just to access notifications?

File: notifications.md [hunk 5]
```diff
@@ -1,60 +1,20 @@
...
 
 1. [Log in to TechBiz portal](log-in-to-TechBiz-portal) using your non-SE GSIB.
 
-2. Click on the bell icon.
+2. Click the bell icon.
+(image)
```
Comment (SenthilKumarSivakami) [inline notifications.md:13]: can move the placeholder to the same step.

File: notifications.md [hunk 6]
```diff
@@ -1,60 +1,20 @@
...
 The list of alert notifications is displayed.
 
-3. Click on a pending action item.
+3. Click **pending action item**.
```
Comment (SenthilKumarSivakami) [inline notifications.md:16]: will there be only one pending action item? no right, so update as needed.

File: notifications.md [hunk 7]
```diff
@@ -1,60 +1,20 @@
...
+
+4. You will be directed to complete the action.
```
Comment (SenthilKumarSivakami) [inline notifications.md:18]: this is not a step. This is an outcome of step 3.

File: notifications.md (hunk 2 above)
Comment (kpslp) [reply notifications.md:5]: i've added this in the next pr. pls help me see if it makes sense.

File: notifications.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply notifications.md:3]: Huda, I am unable to see the changes you made while resolving.

File: notifications.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply notifications.md:5]: what do you mean by next PR?

**Review by SenthilKumarSivakami — APPROVED** [review]

**Review by lamshiling — APPROVED** [review]

**Pull request merged**